src/pages/index.js

```diff
@@ -28,6 +28,17 @@ const ColSubheader = styled.h3`
   font-size: 1.4rem;
   font-weight: 600;
 `;
+const SignForm = styled.iframe`
+  height: 912px;
+  @media (max-width: 700px) {
+    height: 970px
+  }
+  @media (max-width: 375px) {
+    height: 1045px
```

Comment:
Sorry, a little confused, maybe because I haven't developed for mobile in a bit -- the height goes up as the viewport gets narrower? is this implicitly responding to pixel density? If so, can we change the media queries to [`pixel-ratio` / `min-resolution`](https://css-tricks.com/snippets/css/retina-display-media-query/) instead of pixel width?

Reply:
As the width gets smaller, the height expands to fit the same content (due to word wraps, and the like).  Otherwise we get a 2nd scrollbar.  As for pr resolution media queries, I haven't seen anyone use them unless you want to specifically target retina stylings (which aren't an issue here).  `px` is not actually a device's hardware pixels.  

Check out https://juiceboxinteractive.com/blog/a-pixel-is-not-a-pixel-designing-for-a-new-generation-of-mobile-devices/

Reply:
I pulled this and played with this, and it makes sense - the content gets longer as it gets narrower, because there are more lines of text, so the frame has to be longer. It's not a pixel resolution issue, it's a text-flow issue.

Might be worth a comment, but in the interests of time I feel like we can merge this now and fine-tune later if needed.